evm/src/fixed_recursive_verifier.rs

```diff
@@ -1022,32 +1007,10 @@ where
         block_inputs
             .set_verifier_data_target(&self.block.cyclic_vk, &self.block.circuit.verifier_only);
 
-        set_block_hashes_target(
```

Comment:
Are these changes supposed to be part of this PR? I'm confused how these many function calls being replaced by one call to `set_public_value_targets`, when there aren't any changes to `set_public_value_targets` itself

Reply:
Indeed that's unrelated, it's just that the public values method for setting target includes all these internally, so while addressing the return values to output a ’Result’ I also cleaned this up. I can revert and address it into a subsequent PR if you prefer!

Reply:
Oh no I think that's fine, since it's a small change! I was just checking